tasks/common/apiHelper.ts

```diff
@@ -413,3 +427,13 @@ function makePackageEntry(pack: string, i: number): string
     return i.toString() + "_" + path.basename(pack);
 }
 
+/**
+ * Adds attachment to Summary tab on VSRM release details
+ */
+export function attachSubmissionSummary(appName: string, submissionUrl: string, submissionResource: any, flightName?: string) {
```

Comment:
Do we need to differentiate between submissionResource and flightName? It looks like when the method is called in the flights, we do flightResource.FriendlyName, and the resource is not really used in the method other than to get the friendlyName. Might be better if we just pass the string of the submisisonName and let the caller do the logic from where to take this value.